conformity/fields/basic.py

```diff
@@ -99,7 +105,7 @@ def _repr(cv):
             self._error_message = 'Value is not one of: {}'.format(', '.join(sorted(_repr(v) for v in self.values)))
 
     def errors(self, value):  # type: (AnyType) -> ListType[Error]
-        if value not in self.values:
+        if not isinstance(value, HashableClass) or value not in self.values:
```

Comment:
Why `isinstance` rather than `try/except TypeError`?

Reply:
Modified to use `try`/`except` instead!